DSharpPlus/Clients/BaseDiscordClient.cs

```diff
@@ -249,6 +249,17 @@ internal bool TryGetCachedUserInternal(ulong user_id, out DiscordUser user)
             return false;
         }
 
+        internal DiscordUser UpdateUserCache(DiscordUser newUser)
+        {
+            return this.UserCache.AddOrUpdate(newUser.Id, newUser, (id, old) =>
```

Comment:
Missing banner and accent color

Reply:
As we discussed on discord, those fields are only sent on `GetUserAsync`, so it's only cached on that.